finetune/input_pipeline.py

```diff
@@ -353,7 +346,8 @@ def get_train_input_fns(self, Xs, Y=None, batch_size=None, val_size=None, contex
         else:
             self._post_data_initialization(Y=None)
 
-        if callable(Xs) or Y is None:
+        if callable(Xs):
```

Comment:
So if Y is none and not a callable (eg normal language model training) we now hit the next block where we try to shuffle None